Frontend/ui-customer/src/Components/StepsHeaderModel/Controller.purs

```diff
@@ -0,0 +1,14 @@
+module Components.StepsHeaderModel.Controller where
+
+import Screens.Types (StepsHeaderModelState)
+import Merchant.Utils as MU
+import Common.Types.App (LazyCheck(..)) as Lazy
+
+data Action = OnArrowClick
+
+stepsHeaderData :: Int -> StepsHeaderModelState
+stepsHeaderData currentIndex = {
+    activeIndex : currentIndex,
+    textArray : ["Let’s get you trip-ready!", "Got an OTP?", "Just one last thing"],
```

Comment:
Handle translations

Reply:
We don't have any language selection in onBoarding screens